packages/datagrid/src/components/DefaultHeaderRenderer/DefaultHeaderRenderer.scss

```diff
@@ -10,6 +10,18 @@ $td-grip-resize-width: 6px !default;
 	}
 }
 
+.td-header-type {
+	&::first-letter {
+		text-transform: uppercase;
+	}
+	text-transform: lowercase;
+
+	// keep height if empty with a Zero-width space character
+	&:empty:before {
+		content: '\200b';
+	}
```

Comment:
Can you set a defined height instead of injecting a char to define space ?

Reply:
no it is more clear to add an empty char that an height

Reply:
Hmm, nope.
When you define a size, to set a size. If you want a box to be 50px height, you set the height, you don't inject 5 invisible chars + line break.
Please just set the size if you want to set a size.

Reply:
we don't have size defined for this element in frontify. I prefer use native height. so i add an zero width if empty to get the same height that there is some char in.

Reply:
I added change the &:empty:before to avoid that the text is centered in the button. It is a behavior for the button. So, i custom the button to show on the top with a display: flex. This avoid to set a height on the subtitle.